src/components/ButtonSortBy/ButtonSortBy.tsx

```diff
@@ -0,0 +1,30 @@
+import { FC } from 'react';
+import cn from 'classnames';
+
+import { TButtonInfo, TOnSortByHandler } from '../../types/SortByButtons';
+
+type TSortByProps = {
+  buttonInfo: TButtonInfo;
+  onSortByHandler: TOnSortByHandler;
+  sortBy: string;
+};
+
+export const ButtonSortBy: FC<TSortByProps> = ({
```

Comment:
This component can be called more beautiful
```suggestion
export const SortButton: FC<TSortButton> = ({
```